### PR TITLE
fix invalid elrond schema

### PR DESCRIPTION
**Pull request by adrianperreault**

### CODE REVIEW

**Review by noslav — APPROVED** [review]
I just reformatted this using the previous https://jsonformatter.curiousconcept.com/# formatting tool to obtain the exact diff! We can proceed with the same.

File: codec/block-elrond.avsc [hunk 1]
```diff
@@ -31,59 +31,55 @@
...
-                                    "name":"Timestamp",
-                                    "type":"long"
-                                 },
-                                 {
-                                    "name":"TxHashes",
-                                    "type":{
-                                       "type":"array",
-                                       "items":"bytes"
-                                    }
+                  "type":[
```
Comment (noslav) [inline codec/block-elrond.avsc:34]: Ah! The redundant `type` was breaking the schema!

**Comment by noslav** [comment]
Step - `Check running agent`
```
bsp-agent          | time="2022-06-07T18:51:05Z" level=error msg="error in connecting to websocket /block dial: dial tcp 34.66.210.112:20003: connect: connection refused" function=consumeWebsocketsEvents line=51
bsp-agent          | time="2022-06-07T18:51:05Z" level=info msg="Connecting to websocket: ws://34.66.210.112:20003/acknowledge" function=consumeWebsocketsEvents line=60
bsp-agent          | time="2022-06-07T18:51:05Z" level=error msg="error in connecting to websocket /ack dial: dial tcp 34.66.210.112:20000: connect: connection refused" function=consumeWebsocketsEvents line=63
bsp-agent          | panic: runtime error: invalid memory address or nil pointer dereference
bsp-agent          | [signal SIGSEGV: segmentation violation code=0x1 addr=0xc0 pc=0xa8e97d]
bsp-agent          |
bsp-agent          | goroutine 51 [running]:
bsp-agent          | github.com/gorilla/websocket.(*Conn).NextReader(0x0)
bsp-agent          | 	/go/pkg/mod/github.com/gorilla/websocket@v1.5.0/conn.go:1000 +0x1d
bsp-agent          | github.com/gorilla/websocket.(*Conn).ReadMessage(0x58)
bsp-agent          | 	/go/pkg/mod/github.com/gorilla/websocket@v1.5.0/conn.go:1093 +0x19
bsp-agent          | github.com/covalenthq/bsp-agent/internal/node.(*elrondAgentNode).consumeWebsocketsEvents.func3()
bsp-agent          | 	/build/internal/node/elrond_node.go:76 +0xc5
bsp-agent          | created by github.com/covalenthq/bsp-agent/internal/node.(*elrondAgentNode).consumeWebsocketsEvents
bsp-agent          | 	/build/internal/node/elrond_node.go:73 +0x56f
bsp-agent exited with code 0
```

**Comment by noslav** [comment]
https://github.com/covalenthq/bsp-agent/runs/6781503665?check_suite_focus=true still failing with the same on latest?

**Comment by adrianperreault** [comment]
> https://github.com/covalenthq/bsp-agent/runs/6781503665?check_suite_focus=true still failing with the same on latest?
https://github.com/covalenthq/bsp-agent/runs/6806887775?check_suite_focus=true Ran it last night and the sockets are now connecting as expected. 👍

**Review by noslav — APPROVED** [review]

**Pull request merged**